instances/latest/terminologies/UBERONParcellation/instances/latest/terminologies/UBERONParcellation/paraventricularNucleusOfTheHypothalamus.jsonld

```diff
@@ -0,0 +1,29 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/UBERONParcellation/paraventricularNucleusOfTheHypothalamus",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
+  "definition": "'Paraventricular nucleus of the hypothalamus' is a nucleus in the anterior part of the hypothalamus.",
+  "description": "",
+  "interlexIdentifier": "http://uri.interlex.org/base/ilx_0773830",
```

Comment:
use "http://uri.interlex.org/base/ilx_0108501" instead. this is the one that links to the UBERON ID